node-scenarios/Dockerfile

```diff
@@ -16,6 +16,7 @@ COPY node-scenarios/env.sh /root/env.sh
 COPY env.sh /root/main_env.sh
 COPY node-scenarios/run.sh /root/run.sh
 COPY common_run.sh /root/common_run.sh
-COPY node-scenarios/node_scenario.yaml.template /root/kraken/scenarios/node_scenario.yaml.template
-COPY node-scenarios/vmware_node_scenario.yaml.template /root/kraken/scenarios/node_scenario.yaml.template
+#COPY node-scenarios/node_scenario.yaml.template /root/kraken/scenarios/node_scenario.yaml.template
```

Comment:
We might want to uncomment this to support node-scenarios for other cloud providers since the config templates are different from vmware. Thoughts?

Reply:
That makes sense.  I suspected it was needed for others.